content/guides/developer-guides/orchestration/_index.md

```diff
@@ -134,13 +147,16 @@ parameters:
     type: string
     required: false
 runner_type: "python-script"
-```
-### Points to remember:
-    a) For every action/task, there should be an entry point. entry_point is the script to execute this action. This can either be a python script or a shell script. OpenSDS prefers python script for using the rich python libraries, as required
-    b) All the parameters of this actions are detailed here. It should define the 'type' of the parameter, whether this parameter is 'required' or not and if the pramaeter can have some 'default' value
-    c) The action should specify 'runner_type'. Ex: runner_type: "python_script"
+```  
+
+### Points to remember  
+
+1. For every action/task, there should be an entry point. entry_point is the script to execute this action. This can either be a python script or a shell script. OpenSDS prefers python script for using the rich python libraries, as required
+2. All the parameters of this actions are detailed here. It should define the '`type`' of the parameter, whether this parameter is '`required`' or not and if the pramaeter can have some '`default`' value
```

Comment:
Quotations are not required if we are using code highlighting. (repeated in several places)

Reply:
I will remove the quotations.

Reply:
@joseph-v  This is fixed in the latest commit.
You can verify and resolve this.